Comp2024/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -99,7 +99,8 @@ public class RobotContainer
   private final Telemetry                             logger          = new Telemetry(kMaxSpeed);
 
   // The robot's shared subsystems
-  private final HID                                   m_hid           = new HID(m_driverPad.getHID( ), m_operatorPad.getHID( ));
+  private final HID                                   m_hid           =
```

Comment:
See comment in HID constructor, but we don't need to pass those last two parameters when HID is created.